src/main/java/edu/harvard/iq/dataverse/settings/JvmSettings.java

```diff
@@ -91,30 +91,69 @@ public enum JvmSettings {
 
     // PERSISTENT IDENTIFIER SETTINGS
     SCOPE_PID(PREFIX, "pid"),
-    
-    // PROVIDER EZID (legacy) - these settings were formerly kept together with DataCite ones
-    SCOPE_PID_EZID(SCOPE_PID, "ezid"),
-    EZID_API_URL(SCOPE_PID_EZID, "api-url", "doi.baseurlstring"),
-    EZID_USERNAME(SCOPE_PID_EZID, "username", "doi.username"),
-    EZID_PASSWORD(SCOPE_PID_EZID, "password", "doi.password"),
+    PID_PROVIDERS(SCOPE_PID, "providers"),
+    PID_DEFAULT_PROVIDER(SCOPE_PID, "default-provider"),
+    PROVIDERS_DIRECTORY(SCOPE_PID, "provider-implementations-directory"),
+    SCOPE_PID_PROVIDER(SCOPE_PID),
+    PID_PROVIDER_TYPE(SCOPE_PID_PROVIDER, "type"),
+    PID_PROVIDER_LABEL(SCOPE_PID_PROVIDER, "label"),
+    PID_PROVIDER_AUTHORITY(SCOPE_PID_PROVIDER, "authority"),
+    PID_PROVIDER_SHOULDER(SCOPE_PID_PROVIDER, "shoulder"),
+    PID_PROVIDER_IDENTIFIER_GENERATION_STYLE(SCOPE_PID_PROVIDER, "identifier-generation-style"),
+    PID_PROVIDER_DATAFILE_PID_FORMAT(SCOPE_PID_PROVIDER, "datafile-pid-format"),
+    PID_PROVIDER_MANAGED_LIST(SCOPE_PID_PROVIDER, "managed-list"),
+    PID_PROVIDER_EXCLUDED_LIST(SCOPE_PID_PROVIDER, "excluded-list"),
+
+    // PROVIDER DATACITE (legacy - single provider setting)
+    SCOPE_LEGACY_PID_DATACITE(SCOPE_PID, "datacite"),
```

Comment:
Having all these legacy settings is a bummer, but not easily avoidable without modifying the alias config source. Do you feel we should enable non-placeholder lookups for aliases for settings with placeholders?

IMHO the legacy settings should be marked `@Deprecated` and removed with Dataverse 7.